srcpkgs/yaru/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'yaru'
+pkgname=yaru
+version=r28
+revision=1
+archs=noarch
+wrksrc="yaru-$version"
```

Comment:
no need to do this, this is the default value of `wrksrc` `${pkgname}-${version}`